assets/scripts/templates/show-display.handlebars

```diff
@@ -0,0 +1,18 @@
+{{#with photo as |photo|}}
+  <section data-id="{{photo._id}}">
+    <img class="full-image" src="{{photo.url}}" alt="photo unavailable">
+    <h3 data-id="{{photo._id}}">
+      {{photo.name}}
+    </h3>
+    <p> Tags:
+      {{#if photo.tags.[0]}}
+        {{photo.tags}}
```

Comment:
Should merge Jake's PR with tags-display first, then change this line after a rebase